boto/s3/lifecycle.py

```diff
@@ -98,9 +115,11 @@ class FilterSet(object):
     </LifecycleConfiguration>
     """
 
-    def __init__(self, prefix='', tag_list=None):
+    def __init__(self, prefix='', tag_list=None, size_gt=None, size_lt=None):
```

Comment:
Do we need to make it obvious that the sizes are strings and not integers?

Reply:
Thanks. Update https://github.com/cloudian/boto/pull/31/commits/427b7cc8cc7b0bfc08937e6480f9634632ae9ff7